src/compements/AddingProduct.jsx

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line no-unused-vars
+import React from 'react';
+import '../styles/addingProduct.css'
+// eslint-disable-next-line react/prop-types
+function AddingProduct({name, dataValue, dataSet}) {
+
+    return (
+        <div className="product-container">
+            {/* eslint-disable-next-line react/prop-types */}
+            <span>{name}</span>
+            {/* eslint-disable-next-line react/prop-types */}
+            <button type="button" onClick={() => dataSet(dataValue = dataValue + 1)}>+</button>
+            {/* eslint-disable-next-line react/prop-types */}
+            <span> {dataValue} </span>
+            {/* eslint-disable-next-line react/prop-types */}
+            <button type="button" onClick={() => dataSet(dataValue = dataValue - 1)}>-</button>
```

Comment:
Je kan deze knop nu ook in de - gebruiken, dat is niet heel netjes. Wat een oplossing zou zijn dat als dataValue 0 is dat dan de knop disabled wordt :)